### ISSUE TITLE
InputTypeCountry filters by ISO and not Label

### ISSUE THREAD

**Issue by kewitz** [opencollective/opencollective#5524]
**Describe the bug**
When you look up the country list for a specific country, you're actually filtering by ISO code and not by the label it displays.
This is a problem in the case of "Switzerland", where you're intuitively searching for "Swi..." but nothing appears. You need to type "CH" to get that country listed.

**To Reproduce**
1. Go to the New Expense page
2. Click on the Choose Country dropdown
3. Type "Switzerland"
4. Type "CH"

**Expected behavior**
Search country by name.

**Screenshots**
![image](https://user-images.githubusercontent.com/2119706/166764236-a103241c-3e4f-4bb4-ae8c-f402c48df87e.png)

![image](https://user-images.githubusercontent.com/2119706/166764938-5e512ce3-3377-45ea-b528-4dc4c3283ff0.png)

**Comment by Betree** [comment]
I think @SudharakaP is already looing in this regression (it was reported [on Slack](https://opencollective.slack.com/archives/C035S573ZD2/p1651208558348219)). Should be resolved by https://github.com/opencollective/opencollective-frontend/pull/7745
